multicluster/controllers/multicluster/serviceexport_controller.go

```diff
@@ -313,11 +313,18 @@ func (r *ServiceExportReconciler) handleDeleteEvent(ctx context.Context, req ctr
 			Namespace: leaderNamespace,
 		},
 	}
-
+	klog.Infof("leaderNamespace: %v", leaderNamespace) // For resExpNamespaced, To delete
 	// clean up Service/Endpoints kind of ResourceExport in remote leader cluster
 	svcResExportNamespaced := common.NamespacedName(leaderNamespace, svcResExportName)
 	epResExportNamespaced := common.NamespacedName(leaderNamespace, epResExportName)
-	err := remoteCluster.Delete(ctx, svcResExport, &client.DeleteOptions{})
+	resExp := &mcsv1alpha1.ResourceExport{}                                                      // For remoteCluster.Get, To delete
+	resExpNamespaced := types.NamespacedName{Namespace: leaderNamespace, Name: svcResExportName} // For remoteCluster.Get, To delete
+	err := remoteCluster.Get(ctx, resExpNamespaced, resExp)                                      // For testing remoteCluster.Delete, To delete
+	klog.Infof("svc get err: %v", err)                                                           // For logging result of remoteCluster.Get, To delete
+	err = remoteCluster.Delete(ctx, svcResExport, &client.DeleteOptions{})
+	klog.Infof("svc err: %v", err)                         // For logging result of remoteCluster.delete, To delete
+	err = remoteCluster.Get(ctx, resExpNamespaced, resExp) // For testing remoteCluster.Delete, To delete
+	klog.Infof("svc get err: %v", err)                     // For logging result of remoteCluster.Get, To delete
```

Comment:
I know it's debug codes, but do you notice that the err here will overwrite the err returned by `err = remoteCluster.Delete(ctx, svcResExport, &client.DeleteOptions{})`, which means below error check is actually checking the error returned by `remoteCluster.Get(ctx, resExpNamespaced, resExp)`?